templates/fine-tune-GPTJ/README.md

```diff
@@ -0,0 +1,62 @@
+# Fine-Tuning LLMs on Anyscale with DeepSpeed
+
+In this application you will fine tune an LLM - GPTJ. GPT-J is a GPT-2-like causal language model trained on the Pile dataset. This particular model has 6 billion parameters. For more information on GPT-J, click [here](https://huggingface.co/docs/transformers/model_doc/gptj).
+
+The application can be used by developers and datascientists alike.  Developers can leverage simple APIs to run fine tuning jobs on the cluster while data scientists and machine learning engineers can dive in deeper to view the underlying code using familiar tools like JupyterLab notebooks or Visual Studio Code. **These tools enable you to easily adapt this example to use other similar models or your own data**.
+
+## Using this application
+You can use the application in one of two ways:
+
+1. Via the CLI.  Navigate to the "terminal" once started and run the following command:
+```
+anyscale job submit -- python gptj_deepspeed_fine_tuning.py
+```
+Once submitted, you can navigate to the Job page and view the training progress with the Ray Dashboard. 
+![Ray Dashboard](https://github.com/anyscale/templates/releases/download/media/raydash.png)
+
+2. By using the interactive notebook which will go step by step through the process of loading data, tokenizing and splitting the data, loading the model, and performing fine tuning.  You can follow along in VSCode or Jupyter!   Note: You can also use the CLI command directly from the interactive notebook.  Take a look under the "Submitting an Anyscale Job" section of the notebook.
+![IDES](https://github.com/anyscale/templates/releases/download/media/ides.png)
+
+### Next Steps
+
+#### Training on your own data: Modifying the Script 
+Once your application is ready and launched you may view the script with VSCode or Jupyter and modify to use your own data!  Read more about loading data with Ray [from your file store or database here](https://docs.ray.io/en/latest/data/loading-data.html).  Make sure the data you use has a similar structure to the [Shakespeare dataset we use.](https://huggingface.co/datasets/tiny_shakespeare)
+
+The lines of code to update look like the following:
+![Code](https://github.com/anyscale/templates/releases/download/media/code.png)
```

Comment:
Replace this with `"Modify the code under the 'loading data' section of the script to load your own fine-tuning dataset."`

Want to avoid needing to update this image 😄